release/trigger-branch

```diff
@@ -1,2 +1,2 @@
-VERSION=1.19
-STEP=1
\ No newline at end of file
+VERSION=1.20
```

Comment:
Fine with updating this, but one can simply set VERSION on the command line before calling the branch.sh. The hope was that we would have automation like we do for the build, so this was the way to set the version. Unfortunately, the automation never occurred.